### PR TITLE
doc: new theme

### PR DESCRIPTION
**Pull request by blueyed**
Via https://github.com/stevedonovan/LDoc/pull/287.

2017-10-21-212800_1251x1045_scrot

2017-10-21-214008_1251x1045_scrot

### CODE REVIEW

File: docs/ldoc.css [hunk 1]
```diff
@@ -0,0 +1,298 @@
+body { 
+    color: #47555c;
+    font-size: 16px;
+    font-family: "Open Sans", sans-serif;
```
Comment (actionless) [inline docs/ldoc.css]: why not just 'Sans' to use the system font?

and the same for monospace above

Comment (blueyed) [reply docs/ldoc.css]: Makes sense to me.
Should be forwarded to https://github.com/stevedonovan/LDoc/pull/287 then.

Comment (actionless) [reply docs/ldoc.css]: copied the comment there

Comment (blueyed) [reply docs/ldoc.css]: Thanks.

Does 9480706 address your comment?

**Comment by blueyed** [comment]
Canceled the build, so ignore the failure from there, please.

**Comment by blueyed** [comment]
Will merge this soon also - we can then improve it from there, and/or roll it back altogether.

**Comment by Elv13** [comment]
@blueyed Merge this if you feel it's ready, otherwise I will press the button later today.

**Pull request merged**